api/src/main/java/org/apache/cloudstack/api/BaseUpdateTemplateOrIsoCmd.java

```diff
@@ -77,6 +79,11 @@ public abstract class BaseUpdateTemplateOrIsoCmd extends BaseCmd {
             description = "optional boolean field, which indicates if details should be cleaned up or not (if set to true, details removed for this resource, details field ignored; if false or not set, no action)")
     private Boolean cleanupDetails;
 
+    @Parameter(name = ApiConstants.ARCHITECTURE, type = CommandType.STRING,
```

Comment:
Should this be amd64 & arm64? Or, name them as: `x86_64 and aarch64`?

Reply:
See outputs of /usr/bin/arch & ` dpkg --print-architecture`.
On x86 machine, arch returns 'x86_64' but dpkg returns 'amd64'.
On rpi5/arm machine, arch returns `aarch64` but dpkg returns `arm64`

Reply:
Thanks @rohityadavcloud - I think its better to stick with the /usr/bin/arch return values as dpkg is only available on debian based OS